project_recalculate/README.rst

```diff
@@ -0,0 +1,123 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+    :target: http://www.gnu.org/licenses/agpl.html
+
+===================
+Project Recalculate
+===================
+
+This module recalculates Task start/end dates depending on Project
+start/end dates.
+
+
+Instalation
+===========
+
+This addon requires Odoo v8 after 2015-08-24 because is not fully functional
+without this patch: `Odoo PR #8208 <https://github.com/odoo/odoo/pull/8208>`
+
+
+Configuration
+=============
+
+You can define working calendar at Setting > Technical > Resource > Working time
+Then assign this calendar to a resource (related with an user) or a project
+
+When calculating task dates, this addon will look for a working calendar in this order:
+
+* If project has working time assigned, use it.
+* If user assigned, search first resource related with this user
+  (normally user is related, only with one resource) and get its working calendar
+  ends at 18:00
+* If not user assigned or resource hasn't working calendar, search first
+  working calendar of the company
+* If no working calendar found, then every day is workable and work starts at
+  08:00 and ends at 18:00
+
+Also you can define which task stages are included in recalculation when
+'Project recalculate' button is clicked. By default, all are included.
+To change this go to Project > Configuration > Stages > Task Stages and change
+'Include in project recalculate' field
+
+
+Usage
+=====
+
+There are two calculation modes:
+
+* **Date begin**: Task start/end dates are recalculated from project's date begin
+* **Date end**: Task start/end dates are recalculated from project's date end
+
+If the project start/end date is changed in the form view, then you have to
+click "Recalculate project" button to recalculate all open tasks [1]
```

Comment:
Update this text to indicate that the affected tasks are those which stage has "Include in project recalculate" flag checked.